### PR TITLE
added Daniel line

### PR DESCRIPTION
**Pull request by danielchegwidden**
Added a line for me :)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM node:16
```
Comment (Booligoosh) [inline Dockerfile:1]: docker

**Comment by Booligoosh** [comment]
Looks good 😛

**Review by Booligoosh — APPROVED** [review]

File: src/__snapshots__/App.test.js.snap [hunk 2]
```diff
@@ -20,6 +20,12 @@ exports[`page renders correctly 1`] = `
     </p>
     <p>
       Ethan Moliver | CMD | Spicy char kway teow | Community
+<<<<<<< HEAD
```
Comment (Booligoosh) [inline src/__snapshots__/App.test.js.snap]: Actually looks like you left one of the merge conflicts in

**Review by Booligoosh — CHANGES_REQUESTED** [review]

File: src/__snapshots__/App.test.js.snap (hunk 2 above)
Comment (danielchegwidden) [reply src/__snapshots__/App.test.js.snap]: yeah made an error there

Comment (danielchegwidden) [reply src/__snapshots__/App.test.js.snap]: should be all good now

**Review by skittleBeans — APPROVED** [review]
Looks G

**Review by Booligoosh — APPROVED** [review]
Looks good now!

**Pull request merged**